releasenotes/notes/appsec-8f19fc73e7c70e7c.yaml

```diff
@@ -0,0 +1,7 @@
+---
+prelude: >
+    Adding AppSec based on a simple gateway
```

Comment:
```suggestion
```
cc @Kyle-Verhoog - feel free to apply this one depending on how we want to advertise :)